x11-packages/lyx/build.sh

```diff
@@ -1,4 +1,4 @@
-TERMUX_PKG_HOMEPAGE=http://www.lyx.org
+TERMUX_PKG_HOMEPAGE=https://www.lyx.org
```

Comment:
`http://www.lyx.org` redirects to `https://www.lyx.org`.
Using HTTPS in the link still makes sense, but isn't going to change anything.